### PR TITLE
remove import / export handler

### PR DESCRIPTION
**Pull request by cgardens**
## What
* The previous PR removed access to import / export in the API to get it out of the hands of users faster. This PR, removes all of the code that supported those features.

## To do
- [x] remove env variables for import / export
- [x] remove ttl file watcher?

## How
* Kill `ArchiveHandler`. Only used for import / export.
* Kill `ConfigDumpImporter` and `ConfigDumpExporter`. Mostly used for import / export. One static function was moved out of there.

## Recommended reading order
1. `airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java`
2. the rest...

### CODE REVIEW

File: airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java [hunk 1]
```diff
@@ -27,10 +27,7 @@
 public class JsonSecretsProcessor {
 
   @Builder.Default
-  final private boolean maskSecrets = true;
-
-  @Builder.Default
-  final private boolean copySecrets = true;
+  final private Boolean copySecrets = false;
```
Comment (cgardens) [inline airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java:30]: @benmoriceau this seems like the safer default, no?

File: airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java [hunk 2]
```diff
@@ -27,10 +27,7 @@
 public class JsonSecretsProcessor {
 
   @Builder.Default
-  final private boolean maskSecrets = true;
```
Comment (cgardens) [inline airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java:30]: @benmoriceau this was only used for the export, right? that was the only usage I could see, but lmk if you think there might be others.

Comment (benmoriceau) [reply airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java:30]: > @benmoriceau this was only used for the export, right? that was the only usage I could see, but lmk if you think there might be others.

I need to double check. I remember a PR from Pedro using it

Comment (benmoriceau) [reply airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessor.java:30]: Yeah, it is ok to remove. Masking the secret should be the default.

**Review by benmoriceau — APPROVED** [review]

**Review by davinchia — APPROVED** [review]

**Review by krishnaglick — APPROVED** [review]
If you merge in master and re-lint `mockDestinationDefinition.json` I think the change will go away.
Meant to comment, not approve. 🤦

**Pull request merged**